coulomb-cats/src/main/scala/coulomb/cats/package.scala

```diff
@@ -0,0 +1,24 @@
+package coulomb.cats
+
+import cats._
+import cats.implicits._
+import coulomb.Quantity
+
+trait CatsImplicits {
+  /**
+    * Eq[Quantity[A, B]] when there is an Eq[A]
+    *
+    * @group typeclass
+    */
+  implicit def eqQuantity[A: Eq, B]: Eq[Quantity[A, B]] = Eq.by(_.value)
+
+  /**
+    * Order[Quantity[A, B]] when there is an Order[A]
+    *
+    * @group typeclass
+    */
+  implicit def orderQuantity[A: Order, B]: Order[Quantity[A, B]] = Order.by(_.value)
```

Comment:
So I take it that the unit type `B` is assumed the same in these comparisons?   `Quantity[Float, Meter] < Quantity[Float, Meter]` is safe to compare just by extracting `value` but isn't correct if the units are different

Reply:
Hmm, that's right. I guess this will only work for example if you're sorting a list with the same unit. Since `B` is `invariant` that should be ok right?

Reply:
I think it is OK as long as it's enforced that both LHS and RHS are same value type and unit type. Currently Quantity is invariant.  I have considered what it might mean to make it covariant (#65) but that is very speculative.

This is the main reason I never defined scala `Ordering` - supporting convertable units (e.g. comparing a quantity of meters with a quantity of yards) doesn't really fit in the scheme of "normal" ordering-related typeclasses.